app/api/currentuser/get_group_memberships_history.feature

```diff
@@ -0,0 +1,208 @@
+    Given I am the user with ID "2"
+    When I send a GET request to "/current-user/group-memberships-history"
+    Then the response code should be 200
+    And the response body should be, in JSON:
+    """
+    [
+      {
+        "id": "19",
+        "group": {
+          "name": "Another Club",
+          "type": "Club"
+        },
+        "status_date": "2016-09-29T06:38:38Z",
+        "type": "left"
+      },
+      {
+        "id": "18",
+        "group": {
+          "name": "Another Team",
+          "type": "Team"
+        },
+        "status_date": "2016-08-29T06:38:38Z",
+        "type": "removed"
+      },
+      {
+        "id": "17",
+        "group": {
+          "name": "Another Class",
+          "type": "Class"
+        },
+        "status_date": "2016-07-29T06:38:38Z",
+        "type": "requestAccepted"
+      },
+      {
+        "id": "16",
+        "group": {
+          "name": "Other people",
+          "type": "Other"
+        },
+        "status_date": "2016-06-29T06:38:38Z",
+        "type": "invitationAccepted"
+      },
+      {
+        "id": "15",
+        "group": {
+          "name": "Our Friends",
+          "type": "Friends"
+        },
+        "status_date": "2016-05-29T06:38:38Z",
+        "type": "requestRefused"
+      },
+      {
+        "id": "14",
+        "group": {
+          "name": "Our Club",
+          "type": "Club"
+        },
+        "status_date": "2016-04-29T06:38:38Z",
+        "type": "requestSent"
+      },
+      {
+        "id": "13",
+        "group": {
+          "name": "Our Team",
+          "type": "Team"
+        },
+        "status_date": "2016-03-29T06:38:38Z",
+        "type": "invitationRefused"
+      },
+      {
+        "id": "12",
+        "group": {
+          "name": "Our Class",
+          "type": "Class"
+        },
+        "status_date": "2016-02-29T06:38:38Z",
+        "type": "invitationSent"
+      }
+    ]
+    """
+
+  Scenario: Request the first row
+    Given I am the user with ID "1"
+    When I send a GET request to "/current-user/group-memberships-history?limit=1"
+    Then the response code should be 200
+    And the response body should be, in JSON:
+    """
+    [
+      {
+        "id": "9",
+        "group": {
+          "name": "Another Club",
+          "type": "Club"
+        },
+        "status_date": "2017-09-29T06:38:38Z",
+        "type": "left"
+      }
+    ]
+    """
+
+  Scenario: Request the first row starting from some date
+    Given I am the user with ID "1"
+    When I send a GET request to "/current-user/group-memberships-history?limit=1&from.status_date=2017-07-29T06:38:38Z&from.id=7"
+    Then the response code should be 200
+    And the response body should be, in JSON:
+    """
+    [
+      {
+        "id": "6",
+        "group": {
+          "name": "Other people",
+          "type": "Other"
+        },
+        "status_date": "2017-06-29T06:38:38Z",
+        "type": "invitationAccepted"
+      }
+    ]
+    """
```

Comment:
no test for the sort arg given?

Reply:
maybe the case where there is nothing after the last notif date?

Reply:
added both tests